RIN.WebAPI/Controllers/ClientApiV1.cs

```diff
@@ -130,6 +130,25 @@ public async Task<object> Delete(long characterGuid)
             }
         }
 
+        // TODO: Game is pushing Key and Namespace through QueryStrings and then returning it, why?
+        // TODO: What is Value and where does it come from?
+        [HttpGet("characters/{characterGuid}/data")]
+        [R5SigAuthRequired]
+        public object Data(long characterGuid)
```

Comment:
This should work to avoid the ``HttpContext.Request.Query``

```csharp
public Task<CharacterDataResp> Data(long characterGuid, [FromQuery] key, [FromQuery] namespace)
{
            var characterData = new CharacterDataResp
            {
                Key = key,
                Namespace = ns,
                Value = "0,1,2,3"
            };

            return characterData;
}
```